pi_config_files/smb.conf

```diff
@@ -235,7 +235,7 @@ guest account = pifinder
 # to the drivers directory for these users to have write rights in it
 ;   write list = root, @lpadmin
 [shared]
-path=/home/pifinder/PiFinder_data
+path=$HOME/PiFinder_data
```

Comment:
I'm not sure $HOME will work in this context as the samba service is not running as the `pifinder` user.  Perhaps the same trick you used to inject the current user into the service definition file would work here?